Gopkg.lock

```diff
@@ -2301,6 +2301,7 @@
     "k8s.io/kubernetes/pkg/apis/core/validation",
     "k8s.io/kubernetes/pkg/capabilities",
     "k8s.io/kubernetes/pkg/client/metrics/prometheus",
+    "k8s.io/kubernetes/pkg/kubectl",
```

Comment:
How is there a change to `Gopkg.lock` without any change to vendor files?

Reply:
I do see that those files are already in vendor.

Reply:
i had originally vendored some additional stuff from k8s.io/kubernetes/pkg/kubectl, but ended up not using it, so when i re-vendored, this is what was left over. i'm quite confused, but each current run of 'dep ensure' keeps that change in there.